jwst/extract_1d/extract.py

```diff
@@ -2973,9 +2999,12 @@ def do_extract1d(
             if source_type is None:
                 source_type = "UNKNOWN"
 
+            if ifu_set_srctype is not None:
+                source_type = ifu_set_srctype
+                log.info('Overriding source type and setting it to POINT')
```

Comment:
the log message should not be hardwired to say "POINT", because the user can set it to either POINT or EXTENDED